docs/section6/Rule6-5.md

```diff
@@ -45,38 +45,23 @@
 
       - For each space in zone: `space_b in zone_b.spaces:`  
 
-        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_space_schedules(space_b.interior_lighting)`  
+        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_interior_lighting_schedules(space_b.interior_lighting, false)`  
 
         - Get matching space in P_RMR: `space_p = match_data_element(P_RMR, Spaces, space_b.id)`  
 
-          - Get normalized space lighting schedule in P_RMR: `normalized_schedule_p = normalize_space_schedules(space_p.interior_lighting)`
+          - Get normalized space lighting schedule in P_RMR: `normalized_schedule_p = normalize_interior_lighting_schedules(space_p.interior_lighting, true)`
```

Comment:
During unoccupied hours schedules should be the same between baseline and proposed, so I think normalized_schedule_p = normalize_interior_lighting_schedules(space_p.interior_lighting, **false**)

Reply:
That is correct. Thanks for pointing it out!